addons/info/src/components/markdown/code.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { SyntaxHighlighter } from '@storybook/components';
+import { ThemeProvider, convert } from '@storybook/theming';
 
-// XXX: is this a bug? should it be (props) => ?
-const Code = ({ props }) => <SyntaxHighlighter bordered copyable {...props} />;
+const Code = ({ code, language, ...props }) => (
+  <ThemeProvider theme={convert()}>
```

Comment:
what does this `convert()` do?

Reply:
this is to make sure we get theme object that's needed by `SyntaxHighlighter`

I also have mixed feeling about this because currently user will be forced with light theme (calling `convert` without argument will make it return light theme)

If we have a way to get currently used theme inside this `Code` component, we can just pass it to `convert` as args